openreview/venue_request/venue_request.py

```diff
@@ -1423,17 +1423,23 @@ def setup_request_form(self):
                 'default': '2',
                 'order': 41
             },
+            'submission_license': {
+                'value-radio': ['CC BY 4.0', 'CC BY-SA 4.0', 'CC BY-NC 4.0', 'CC BY-ND 4.0', 'CC BY-NC-SA 4.0', 'CC BY-NC-ND 4.0', 'CC0 1.0'],
+                'description': 'Which license would you like to use for each submission? If you are unsure, we recommend "CC BY 4.0". If your license is not listed, please let us know and we can add it.',
```

Comment:
@celestemartinez should we mention the link to the "Article License" in openreview term of use?

Reply:
Yes! I don't think we can link to the specific section, right? We can have something like:

> Please refer to https://openreview.net/legal/terms for more information.

Reply:
Updated. This is what it looks like:

<img width="717" alt="Screenshot 2023-10-11 at 4 22 49 PM" src="https://github.com/openreview/openreview-py/assets/22416602/fa5ca258-b38c-4f06-8170-54d5f5822022">